spiketoolkit/curation/curationsortingextractor.py

```diff
@@ -81,21 +80,21 @@ def exclude_units(self, unit_ids):
         unit_ids: list
             The unit ids to be excluded
         '''
-        if(len(unit_ids) == 0):
+        if len(unit_ids) == 0:
             return
         root_ids = []
         for i in range(len(self._roots)):
             root_id = self._roots[i].unit_id
             root_ids.append(root_id)
 
-        if(set(unit_ids).issubset(set(root_ids)) and len(unit_ids) > 0):
+        if set(unit_ids).issubset(set(root_ids)) and len(unit_ids) > 0:
```

Comment:
Is "set" fatser than np.in1d ?
I personnally find in1d more convinient to read.

Reply:
@colehurwitz wrote this. I think this i quite fast anyways..